azure-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+steps:
+- task: Gradle@2
+  displayName: Build core
+  inputs:
+    gradleWrapperFile: 'gradlew'
+    jdkVersionOption: '1.8'
+    options: '--no-daemon --continue'
```

Comment:
Couldn't we actually use the daemon?

Reply:
I wouldn't, given that it is running on an actual machine that being reused and may cause troubles